### PR TITLE
Updating ResourceManager to invoke WinRT ResourceManager APIs through WinRT callbacks

### PR DESCRIPTION
**Pull request by SedarG**
@tarekgh , @alexperovich PTAL

Description can be found on 3rd commit message. (https://github.com/dotnet/corefx/pull/12631/commits/0ef62572d183ed7319ca85ef7ceb960763bcdbb7)

/cc @dotnet/corert-contrib

### CODE REVIEW

File: dependencies.props [hunk 1]
```diff
@@ -1,18 +1,27 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+
+  <!-- There are situations where the live .Net Native targeting pack wouldn't be suitable to pick up.
+       These are usually temporary, but requires 'freezing' the .net native targeting pack on CoreFX.
+       Use the property below to control this behavior
+  -->
+  <PropertyGroup>
+    <DisableProjectNTfsAutoUpgrade Condition="'$(DisableProjectNTfsAutoUpgrade)'==''">true</DisableProjectNTfsAutoUpgrade>
```
Comment (tarekgh) [inline dependencies.props]: >ProjectN [](start = 12, length = 8)

could you change the name from PojectN to Sonething like NetNative? ProjectN is really internal name

**Comment by tarekgh** [comment]
@SedarG looks the ResourceManager.cs is not included any more. am I am missing something?

File: dependencies.props (hunk 1 above)
Comment (tarekgh) [inline dependencies.props]: @SedarG nevermind then

**Comment by alexperovich** [comment]
On coreclr ResourceManager is not part of corelib. I didn't do anything for uap10.1.

**Comment by SedarG** [comment]
> @SedarG looks the ResourceManager.cs is not included any more. am I missing something?
ResourceManager.cs, in this location, is for .net native, it's being included in uap101aot configuration.

**Comment by tarekgh** [comment]
LGTM

**Comment by SedarG** [comment]
@dotnet-bot test please

**Pull request merged**